docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -51,11 +51,11 @@ General-purpose aggregation functions
 
         Examples::
 
-            "some_field"
-            "-some_field"
             from django.db.models import F
 
-            F("some_field").desc()
+            SomeModel.objects.aggregate(arr=ArrayAgg("a_field")).order_by("-some_field")
+
+            SomeModel.objects.aggregate(arr=ArrayAgg("a_field")).order_by(F("some_field").desc())
```

Comment:
`order_by` here is a kwarg or `ArrayAgg` I think:

```suggestion
SomeModel.objects.aggregate(arr=ArrayAgg("a_field", order_by="-some_field"))

SomeModel.objects.aggregate(arr=ArrayAgg("a_field", order_by=F("some_field").desc()))
```

Reply:
This was fixed, thank you! Great catch :star2: